packages/crayons-core/src/components/select/select.tsx

```diff
@@ -646,6 +663,45 @@ export class Select {
     else if (this.state === 'warning') return `warning-${this.name}`;
     return null;
   }
+
+  onPaste(e) {
+    // Allow paste only if isCreatable is true
```

Comment:
why are we allowing paste only if `isCreatable` is true. Should we enable this `paste` functionality for all the usecases ?

Reply:
@arvindanta paste functionality can contain values which are not part of the predefined options. With search, we will not be able to know the options available to the user. By allowing this only when isCreatable is true, we would allow this functionality to add values which may or may not exist in the options as user might not need this functionality by default and only when user wants to input custom values.